chapters/lang/expressions.tex

```diff
@@ -257,7 +321,30 @@ \section{Conditionals}
 
 A \emph{conditional expression}\index{expression!conditional expression} is a conditional branch in program control. Conditional expression is made of a condition block/expression, followed by a consequent block, any number of alternative conditions and blocks, and an optional trailing final alternative block. If a condition expression evaluates successfully, the consequent block is executed and any subsequent \lstinline{else if} or \lstinline{else} block is skipped. If a condition expression evaluation fails, the consequent block is skipped and any subsequent else if condition is evaluated. If all condition blocks fails then \lstinline{else} block is executed. Result values of conditions in condition block are ignored. Conditional expression evaluates to the same value as the executed block, or succeeded unit if no block is evaluated.
 
-TODO(Ania): Example
+\begin{example}[Trivial if expression]
+\begin{lstlisting}
+  if a < b { c = a } 
+\end{lstlisting}
+\end{example}
+
+The variables definied in condition could be used in ifbraceblock and elsebraceblock as it was the same scope.
+
+\begin{example}[Trivial if expression with else (version1)]
+\begin{lstlisting}
+    if (let a = 3; a < b) { c = a } 
```

Comment:
`let` ?

Reply:
Last time, we decide to make explicit declaration with keyword 'let'. Anything has been changed?